server/routes/api/babysitter/babysitterRoutes.ts

```diff
@@ -38,4 +52,129 @@ babysitterRouter.get(
   }
 );
 
-export default babysitterRouter;
+babysitterRouter.put(
+  "/image/:babysitterId",
+  [
+    param("babysitterId").notEmpty().isNumeric().withMessage(BABYSITTER_INVALID_INPUT_ERROR)
+  ],
+  upload.single('image'),
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitterId } = req.params;
+      const validation = await handler.userValidation(Number(babysitterId));
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+
+      const imageName = `babysitter_${babysitterId}`;
+
+      if (!req.file) {
+        return res.status(400).json({ error: `Missing file input.` });
+      }
+      
+      const response = await s3.putImage(req.file, imageName);
+      if (!response) {
+        return res.status(500).json({ message: `Couldn't upload image` });
+      }
+      await handler.putProfileImage(imageName, Number(babysitterId));
```

Comment:
I would make the handler function call the s3.putImage function.
The handler will first call this function and if error then will return a message.
Then the const response from the handler call will check if gotten a message - if so, then it means there is an error to print